src/build/server.js

```diff
@@ -44,7 +44,7 @@ export async function buildServer(
 ) {
   log.info(false, 'Building server…');
 
-  if (process.env.API_ONLY) {
+  if (process.env.API_ONLY && JSON.parse(process.env.API_ONLY)) {
```

Comment:
Just an idea. Wonder if this process environment would be better replaced by an optional param on buildServer.

If you load the config in https://github.com/wedeploy/magnet/blob/master/src/magnet.js#L108 and pass the apiOnly configuration to the build function (which can default to false) there would be no need for that environment variable.